selfdrive/athena/tests/test_registration.py

```diff
@@ -19,16 +18,11 @@ def setUp(self):
     self.params = Params()
     self.params.clear_all()
 
-    self.persist = tempfile.TemporaryDirectory()
-    os.mkdir(os.path.join(self.persist.name, "comma"))
-    self.priv_key = Path(os.path.join(self.persist.name, "comma/id_rsa"))
-    self.pub_key = Path(os.path.join(self.persist.name, "comma/id_rsa.pub"))
-    self.persist_patcher = mock.patch("openpilot.selfdrive.athena.registration.PERSIST", self.persist.name)
-    self.persist_patcher.start()
+    persist_dir = Path(Paths().persist_root()) / "comma"
```

Comment:
some references use `Paths().` and some use `Paths.` Can we decide on one for consistency?

Reply:
done